src/routes.tsx

```diff
@@ -0,0 +1,14 @@
+import { useSelector } from "react-redux";
+import { Routes, Route } from "react-router-dom";
+import { Login } from "./pages";
+
+export const Router = () => {
+  const { isLogged } = useSelector((state: any) => state.user);
```

Comment:
Please, check the types and change any to State type, if i an not wrong, there is that specified in redux toolit docs
Find for RootStateType or something like this